src/gravel/controllers/config.py

```diff
@@ -56,6 +62,7 @@ class OptionsModel(BaseModel):
     devices: DevicesOptionsModel = Field(DevicesOptionsModel())
     status: StatusOptionsModel = Field(StatusOptionsModel())
     services: ServicesOptionsModel = Field(ServicesOptionsModel())
+    etcd: EtcdOptionsModel = Field(EtcdOptionsModel())
```

Comment:
perhaps should be a standalone `etcd.json` file??

Reply:
Dunno... It's etcd as spawned by Aquarium, so we own it, so it's OK to keep it inside the larger Aquarium config? (I don't really have an opinion)

Reply:
I'd leave it in our own config until there's a strong reason not to.